repository/expression.js

```diff
@@ -1,5 +1,23 @@
 import { User, Expression, UserExpression } from '../models';
 
+export const getAllExpressions = async () => {
+  return await User.findAll({
```

Comment:
Why do you need `return await` here?

Reply:
I dont quite understand what this is doing. I see that you are getting a list of users with their expressions, but why isn't it in the user repo since you are making a query on the users table. Either query expressions and include on each expressions the username of the user or move this in the user repo and maybe rename it `getUsersWithExpressions`

Reply:
If you look at the last commit, you will notice that I don't use return await there. I will try to rewrite the query based on expressions or I will move it.